tools/decdk/deps.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+
+const pkg = require('./package.json');
+const deps = pkg.dependencies || {};
+
+const root = path.resolve('..', '..', 'packages', '@aws-cdk');
+const modules = fs.readdirSync(root);
+let errors = false;
+
+for (const dir of modules) {
+  const module = path.resolve(root, dir);
+  if (!fs.existsSync(path.join(module, '.jsii'))) {
+    continue;
+  }
+  const meta = require(path.join(module, 'package.json'));
+
+  const exists = deps[meta.name];
+  if (!exists) {
+    console.error(`missing dependency: ${meta.name}`);
+    errors = true;
+  }
+
+  const requiremennt = `^${meta.version}`;
```

Comment:
Typo

Reply:
Fixed thanks, and added some description